dolphinscheduler-api/pom.xml

```diff
@@ -164,6 +164,11 @@
             <artifactId>swagger-bootstrap-ui</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-annotations</artifactId>
+        </dependency>
```

Comment:
Seems you only need to remove the `<exclusions>` at line 154 😄

Reply:
> exclusions

OMG... I didn't see this...